### ISSUE TITLE
[VT-7] vit; "R" launches review mode

### ISSUE THREAD

**Issue by taskwarrior** [scottkosty/vit#29]
_David Patrick on 2013-08-12T20:32:13Z says:_ 

 by passing the [filter] set to trev

https://github.com/nocejo/trev

vit changes to Review mode, till you're done with that, and back again.

(trev is an excellent taskwarrior review script by Fidel Mato, you really must try it!)

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2013-08-12T20:32:13Z
Modified: 2014-02-09T01:14:58Z
```

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2013-08-14T07:46:49Z says:_ 

 This seems like another job for shortcuts. We need to allow for the filter to be passed to trev though. What is the specific command that you would run in trev?

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2013-09-15T01:08:55Z says:_ 

 It will be easy to implement this with a keybind in 1.2.

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2013-09-15T06:37:45Z says:_ 

 Fixed at 1da206b8.
